packages/create-vuestic/src/utils/package-manager.ts

```diff
@@ -0,0 +1,25 @@
+export type PackageManagerName = "npm" | "pnpm" | "yarn";
+
+let manager: PackageManagerName
+export const getPackageManagerName: () => PackageManagerName = () => {
+  if (manager) {
+    return manager
+  }
+  // This environment variable is set by npm and yarn but pnpm seems less consistent
+  const userAgent = process.env.npm_config_user_agent
+
+  if (userAgent) {
+    if (userAgent.startsWith("yarn")) {
+      manager = "yarn"
+    } else if (userAgent.startsWith("pnpm")) {
+      manager = "pnpm"
```

Comment:
I saw that nuxt recommends to use `--shamefully-hoist` flag on installing. Maybe when is project is generating will be nice to add `.npmrc` file? Where we can put this flag.

Reply:
Nudt already does it, so we don't need to.